resources/views/dashboard/admin/bronze-mic.blade.php

```diff
@@ -54,19 +55,34 @@
                     <td>
-                        @if($winner == null)
-                            {!! Form::open(['url' => '/dashboard/admin/bronze-mic/'.$year.'/'.$month.'/'.$stats[$h->id]->bronze_hrs.'/'.$h->id]) !!}
-                                @csrf
-                                <button action="submit" class="btn btn-success btn-simple-tooltip" data-toggle="tooltip" title="Set as Winner for <?=$mname?>"><i class="fas fa-check"></i></button>
-                            {!! Form::close() !!}
-                        @elseif($winner->controller_id == $h->id)
-                            <a href="/dashboard/admin/bronze-mic/remove/{{ $winner->id }}/{{ $year }}/{{ $month }}" class="btn btn-danger btn-simple-tooltip" data-toggle="tooltip" title="Remove Winner"><i class="fas fa-times"></i></a>
-                        @else
-                            <p>Winner already selected.</p>
-                        @endif
+                        <div class="btn-group">
+                            @toggle('local-hero')
+                                @if($winner_local == null)
+                                    {!! Form::open(['url' => '/dashboard/admin/local-hero/'.$year.'/'.$month.'/'.$stats[$h->id]->local_hero_hrs.'/'.$h->id]) !!}
+                                        @csrf
+                                        <button action="submit" class="btn btn-primary btn-simple-tooltip" data-toggle="tooltip" title="Set as local hero Winner for <?=$mname?>"><i class="fas fa-trophy"></i></button>
+                                    {!! Form::close() !!}
+                                @elseif($winner_local->controller_id == $h->id)
+                                    <a href="/dashboard/admin/local-hero/remove/{{ $winner_local->id }}/{{ $year }}/{{ $month }}" class="btn btn-danger btn-simple-tooltip" data-toggle="tooltip" title="Remove local hero Winner"><i class="fas fa-times"></i></a>
+                                @else
+                                    <p>Winner already selected.</p>
+                                @endif
+                            @endtoggle
+                            @if($winner == null)
+                                {!! Form::open(['url' => '/dashboard/admin/bronze-mic/'.$year.'/'.$month.'/'.$stats[$h->id]->bronze_hrs.'/'.$h->id]) !!}
+                                    @csrf
+                                    <button action="submit" class="btn btn-simple-tooltip" style="background-color:#C9AE5D" data-toggle="tooltip" title="Set as bronze Winner for <?=$mname?>"><i class="fa fa-microphone"></i></button>
+                                {!! Form::close() !!}
+                            @elseif($winner->controller_id == $h->id)
+                                <a href="/dashboard/admin/bronze-mic/remove/{{ $winner->id }}/{{ $year }}/{{ $month }}" class="btn btn-danger btn-simple-tooltip" data-toggle="tooltip" title="Remove bronze mic Winner"><i class="fas fa-times"></i></a>
```

Comment:
See my note in Discord about changing this styling a little bit to make things easier to see at a glance

Reply:
Done!